packages/core/src/components/tooltip/tooltip.stories.tsx

```diff
@@ -79,12 +79,26 @@ export default {
         },
       },
     },
+    offsetDistance: {
+      name: 'Offset Distance',
+      control: {
+        type: 'number',
+      },
+    },
+    offsetSkidding: {
+      name: 'Offset Skidding',
+      control: {
+        type: 'number',
+      },
+    },
```

Comment:
```suggestion
    offsetDistance: {
      name: 'Offset Distance',
      description: 'Sets the distance between the wrapped component and Tooltip',
      control: {
        type: 'number',
      },
    },
    offsetSkidding: {
      name: 'Offset Skidding',
      description: 'Sets offset to Tooltip's position, moving Tooltip left-right or top-down - depending on the placement prop.'
      control: {
        type: 'number',
      },
    },
```

Reply:
I thought the offsetDistance just moved the tooltip vertically and the skidding moved it horizontally? @theJohnnyMe

Reply:
> I thought the offsetDistance just moved the tooltip vertically and the skidding moved it horizontally? @theJohnnyMe

Try this, it depends on the placment prop:
![Screenshot 2024-02-20 at 10 24 35](https://github.com/scania-digital-design-system/tegel/assets/17004063/0e0b03d3-1d37-42ad-9929-b064a5adb1f0)

Reply:
offsetDistance is like padding around the wrapped component while skidding is moving the moving Tooltip a bit left/right or top/down, depending on the placement prop it seems.